styleguide/derek/global-components/modal/_region-select.scss

```diff
@@ -128,3 +128,9 @@
     column-gap: 15px;
   }
 }
+
+@media only screen and (max-width: $screen-xs-max) {
+  .basement-container {
+    padding-bottom: 35px;
```

Comment:
So looking at production right now the chat button takes up a good 65px or so.  So maybe we just bump the padding to 70px instead?

Reply:
He made it 75px ~ and I think this looks fine.